src/rules/custom.js

```diff
@@ -27,10 +28,11 @@ module.exports = {
          * @param {Cheerio} $ A cheerio representation of the document
          * @param {AST} ast The underlying AST representation of the document.
          *                  This should be given to Reporter when warning/erroring with a node.
+         * @param {Info} info Info related to the current file being linted.
```

Comment:
If it's intended purely for information about the file being linted, would it make more sense to call this parameter `file` (and the type `File`)?

Else, if the intend is to keep it generic, should the object type definition be changed to, e.g.:

```js
/** @typedef {{ file: { path: string } }} Info */
```

or

```js
/** @typedef {{ filepath: string }} Info */
```

Reply:
renamed path -> filepath